server/user_account/views.py

```diff
@@ -145,18 +142,29 @@ def remove_token(self, request): # pylint: disable=no-self-use
                         status=status.HTTP_200_OK)
 
 
-class AccessAllClients(generics.GenericAPIView):
-    queryset = CustomUser.objects.values(
-        'user_name',
-        'first_name',
-        'last_name',
-        'role',
-        'is_active',
-        'email',
-        )
-    http_method_names = ['get']
-
+class AccessAllClients(viewsets.ModelViewSet):
```

Comment:
Is this even being called? I don't see any URL redirecting to it. If not used delete it please.